storages/backends/gcloud.py

```diff
@@ -142,12 +146,19 @@ def get_default_settings(self):
             # roll over.
             "max_memory_size": setting("GS_MAX_MEMORY_SIZE", 0),
             "blob_chunk_size": setting("GS_BLOB_CHUNK_SIZE"),
+            "sa_email": setting("GS_SA_SIGNING_EMAIL")
         }
 
     @property
     def client(self):
+        credentials, project_id = auth.default(scopes=['https://www.googleapis.com/auth/cloud-platform'])
+        credentials.refresh(requests.Request())
+        if not hasattr(credentials, "service_account_email"):
+            credentials.service_account_email = self.sa_email
+        _LOGGER.debug(f"Signing email: {credentials.service_account_email}")
```

Comment:
This will log every time this uncached property is touched.

Reply:
sure removed.